tests/Makefile

```diff
@@ -57,8 +75,9 @@ build-test-binary:
 build-test-binary-docker:
 	docker run -v `pwd`/..:/workspace golang:1.17 sh -c "cd /workspace/tests && make build-test-binary"	
 
-run-test: 
-	VENOM_VAR_MY_ENVAR=foo ./venom_wrapper.sh run -vv --format=xml --output-dir=. --lib-dir=./lib_custom --var='array_from_var=["biz","buz"]' --var-from-file ./kafka/testVariables.yml  --var-from-file ./vars/vars.yml ./*.yml ./assertions/*.yml && \
+run-test: generate-venom-pki
+	VENOM_VAR_MY_ENVAR=foo  ./venom_wrapper.sh run \
+	-vv --format=xml --output-dir=. --lib-dir=./lib_custom --var='array_from_var=["biz","buz"]' --var-from-file ./kafka/testVariables.yml --var-from-file=$(PKI_VAR_FILE) --var-from-file ./vars/vars.yml ./*.yml ./assertions/*.yml && \
```

Comment:
this `--var-from-file=$(PKI_VAR_FILE)` is needed to generate the yaml formatted PEM import file used in the test.